RIOT-OS/RIOT
# tests: factor out common Kconfig configuration for sync

### Contribution description
This adds a common configuration file for all the test applications that have a `tests` folder, which enables the test_sync module.

### Testing procedure
- Green CI. The tests that are already in place should check that the modules used are still the same

### Issues/PRs references
None
